### PR TITLE
docs: add server reference section

### PR DESCRIPTION
**Pull request by stevenGravy**
- adds reference section for server including
    - cli
    - audit
    - sessions
    - config

### CODE REVIEW

File: docs/pages/includes/disable-recording-cluster.mdx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+## Disabling session recording
+
+Session recording is enabled by default, and can be disabled in one of two ways:
```
Comment (zmb3) [inline docs/pages/includes/disable-recording-cluster.mdx:3]: This no longer reads correctly. This text is specific to desktop recordings. I would revert.

**Review by zmb3 — CHANGES_REQUESTED** [review]

File: docs/pages/desktop-access/reference/sessions.mdx [hunk 2]
```diff
@@ -22,7 +22,7 @@ auth_service:
 <Admonition type="warning">
 Disabling session recording at the cluster level applies to all types of
 recordings. For example, this configuration would prevent Teleport from
-recording SSH sessions as well as desktop sessions.
+recording SSH and Kubernetes sessions as well as desktop sessions.
```
Comment (zmb3) [inline docs/pages/desktop-access/reference/sessions.mdx:25]: This is awkward - we say _all types of recordings_ and then we mention 3 specifically. What about app or DB recordings?

Personally, I would have left this as-is.

File: docs/pages/includes/disable-recording-cluster.mdx (hunk 1 above)
Comment (zmb3) [reply docs/pages/includes/disable-recording-cluster.mdx:3]: This still seems incorrect.

We say _can be disabled in one of two ways_ and then we list only one (not two) ways.

Disabling via RBAC is only supported for desktop, so I don't think we should use an include here.

File: docs/pages/includes/reference/cli/tsh-ls.mdx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+# grav-00   52e3e46a-372f-494b-bdd9-a1d25b9d6dec 10.164.0.0:3022    os:linux
+# grav-01   73d86fc7-7c4b-42e3-9a5f-c46e177a29e8 10.156.0.2:3022    os:linux
+# grav-02  24503590-e8ae-4a0a-ad7a-dd1865c04e30 10.156.0.7:3022     os:osx
+
+# Only show nodes with os label set to 'osx':
+$ tsh ls os=osx
+
+# Node Name Address            Labels
+# --------- ------------------ ------
+# grav-02   10.156.0.7:3022    os:osx
```
Comment (zmb3) [inline docs/pages/includes/reference/cli/tsh-ls.mdx:55]:
```suggestion
# Only show nodes with os label set to 'mac':
$ tsh ls os=mac

# Node Name Address            Labels
# --------- ------------------ ------
# grav-02   10.156.0.7:3022    os:mac
```

OS X is legacy terminology, let's avoid it.

File: docs/pages/server-access/reference/audit.mdx [hunk 4]
```diff
@@ -0,0 +1,277 @@
...
+  "time": "2023-07-04T23:32:06.838Z",
+  "uid": "70dbf7aa-b496-4438-ab12-622751eab872",
+  "user": "jeff"
+}
+```
+
+
+## session data (T2006I)
+
+Emitted when a session is updated.
```
Comment (zmb3) [inline docs/pages/server-access/reference/audit.mdx:123]: What does it mean for a session to be updated?

File: docs/pages/server-access/reference/cli.mdx [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+(!docs/pages/includes/reference/cli/tsh-ssh.mdx!)
+
+(!docs/pages/includes/reference/cli/tsh-config.mdx!)
+
+(!docs/pages/includes/reference/cli/tsh-proxy-ssh.mdx!)
+
+(!docs/pages/includes/reference/cli/tsh-recordings-ls.mdx!)
+
+(!docs/pages/includes/reference/cli/tsh-play.mdx!)
```
Comment (zmb3) [inline docs/pages/server-access/reference/cli.mdx:15]: I think we're overusing includes here. This makes it very hard to ensure that the page has a reasonable flow and transitions nicely from one section to the next.

I would use includes for smaller snippets that make sense to be reused in different places, not to write entire docs pages.

File: docs/pages/server-access/reference/sessions.mdx [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+the Management section. Server recordings show a server icon in the first
+column to distinguish them from other recordings.
+
+![Server Session Recording](../../../img/server-access/session-recording@2x.png)
+
+Click the play button to open the player in a new tab. You can play, pause,
+rewind and seek specific points.  While viewing you can copy and paste text.
+
+Each session playback is recorded within the audit log for which user viewed
+the session.
```
Comment (zmb3) [inline docs/pages/server-access/reference/sessions.mdx:22]: An audit event is emitted when a recorded session is accessed.

File: docs/pages/server-access/reference/sessions.mdx [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+To play recordings for viewing from the CLI or exporting use the `tsh play` command:
+
+```code
+$ tsh play <SESSION_ID>
+```
+
+<Admonition type="tip" title="Listing recordings">
```
Comment (zmb3) [inline docs/pages/server-access/reference/sessions.mdx:30]: Feels odd for an admonition to be the last thing on the page.

**Review by zmb3 — COMMENTED** [review]
This still feels like too big a change with overuse of includes.

I would prefer to break this up into a series of smaller less controversial changes.

**Comment by stevenGravy** [comment]
Closing to add with smaller changes

**Pull request closed without merging**